nss-day-cohort-47/tabloid-mvc-reese-s-puffs
# Tl user details

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.
- [ x] New feature (non-breaking change which adds functionality)
- [ x] This change requires a documentation update

1.ALTER TABLE UserProfile
ADD COLUMN Active int;

2.UPDATE UserProfile
SET Active = 1
WHERE id =@id;

Run the code above in order in new query. Run #2 for each user you have in your database adjusting the id as needed.


# Testing Instructions

-Use SQL provided to update UserProfile table.
-UserProfile details should show with image default
-Check that a user can be deactivate on UserProfile Index
-Go to view deactivated users to check that a user can be reactivated


# Checklist:

- [ x] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new warnings or errors
- [x ] I have added test instructions that prove my fix is effective or that my feature works
